test/groovy/util/JenkinsPropertiesRule.groovy

```diff
@@ -45,10 +45,10 @@ class JenkinsPropertiesRule implements TestRule {
     }
 
     static Properties loadProperties(String path) {
-        File configFile = new File(path)
+        def inputStream = new File(path).newInputStream()
         def properties = new Properties()
-        FileInputStream inputStream = new FileInputStream(configFile)
         properties.load(inputStream)
+        inputStream.close()
```

Comment:
The input stream get never closed in case there is an exception thrown from ```properties.load(./.)```. Maybe here is does not hurt so much like in case of a long running server where we do not have any file handles after some period of time, but nevertheless I would like to suggest to use [try-with-resource](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html). Other option would be (here in groovy) to use ```File.withInputStream { s-> ./.}```